staticaddr/loopin/manager.go

```diff
@@ -189,12 +206,217 @@ func (m *Manager) Run(ctx context.Context, currentHeight uint32) error {
 				return ctx.Err()
 			}
 
+		case sweepReq := <-sweepReqs:
+			err = m.handleLoopInSweepReq(ctx, sweepReq)
+			if err != nil {
+				log.Errorf("Error handling loop-in sweep request: %v",
+					err)
+			}
+
 		case <-ctx.Done():
 			return ctx.Err()
 		}
 	}
 }
 
+// notifyNotFinished notifies the server that a swap is not finished by
+// sending an empty signature map.
```

Comment:
"by sending an empty signature map"
This is a part of API.

I propose to add this to description of PushStaticAddressSweeplessSigsRequest.signing_info.

Also, we can make this more explicit by making a oneof: a map or an error (string). If the swap is not finished or any other error happened during signing, we would send PushStaticAddressSweeplessSigsRequest with an error message instead of a map.